### PR TITLE
Mobile section

### PR DESCRIPTION
**Pull request by Salwa99**
adding changes in HTML and CSS files: 
   -change the navbar tag
   -change social icons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,43 +9,58 @@
...
+            flex-direction: column;
+            align-items: flex-start;
+            padding: 114px 24px;
+            gap: 12px;
+            position: absolute;
+            width: 375px;
+            height: 604px;
+            left: calc(50% - 375px/2);
+            top: 92px;
         }
```
Comment (Kamba56) [inline index.html:25]: - [Optional] Please remove all styling from the HTML file. It is best practice to have all your styles in a `.css` file.

- [Optional] Please in all future projects, do not create a new pull request for a re-review. Always implemented suggested changes and request for a re-review using the same pull request.

File: index.html [hunk 2]
```diff
@@ -9,43 +9,58 @@
...
+            <div class="hamburger">
+                <a href="#"><img src="style/img/hamburger-vactors.png" class="hamburger-img"></a>
+            </div>
         </nav>
     </header>
 
-    <section>
-        <h1 class="title">I'm Salwa <br>Glad to see you!</h1>
-        <p class="presentation">I’m a software developer! I can help you build a product , feature or
-            website Look through some of my work and experience! If you like what
```
Comment (Kamba56) [inline index.html:37]: - [Optional] Consider using an indentation of two spaces consistently in all your documents. This is a conventional and also [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

File: style/style.css [hunk 3]
```diff
@@ -6,17 +6,33 @@
   padding: 0;
   box-sizing: border-box;
   text-decoration: none;
-  width: 375px;
-  height: auto;
   position: relative;
   left: 15px;
   top: 15px;
```
Comment (Kamba56) [inline style/style.css:11]: - [Optional] I noticed you added a position relative to all your elements and this is making your whole page to be shifted to the left of the screen. Please consider adding the position relative to a class and assign that class to any element you want to instead. For example;
```css
.rel {
  position: relative;
  left: 15px;
  top: 15px;
}
```
Then in your HTML;
```html
<div class="rel">
  ...
</div>
```

**Review by Kamba56 — APPROVED** [review]
Hi @Salwa99, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] All linters passed!👏👏👏
- [x] Great use of Github flow!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Salwa99** [comment]
@Kamba56 Thank you so much for your notice, I will add those changes.

**Pull request merged**